cfpb/owning-a-home
# Monthly payment worksheet: outputs

Adds live outputs to worksheet

## Additions

- Adds `worksheet-output.jsx`, a worksheet-specific output component that displays properties taken or calculated from worksheet data
- Adds `output-usd.jsx`, which displays values formatted as usd
- Adds remaining sections to worksheet

## Changes

- Updates styling of worksheet

## Testing

1. Navigate to [monthly payment worksheet](http://localhost:7000/monthly-payment-worksheet/) locally
2. Verify that worksheet state values from `monthly-payment-worksheet.js` are output correctly in page

## Review

- @contolini 
- @cfarm 

## Screenshots

![outputs](https://cloud.githubusercontent.com/assets/778171/10544839/8c737668-73da-11e5-9800-075ff807492d.png)


## Todos

- Run calculations outside of worksheet output components
- Tests for components

## Checklist
